pkg/homedir/homedir.go

```diff
@@ -6,11 +6,16 @@ import (
 	"path/filepath"
 )
 
-// GetDataHome returns XDG_DATA_HOME.
-// GetDataHome returns $HOME/.local/share and nil error if XDG_DATA_HOME is not set.
+// DataHome (deprecated)
```

Comment:
For all these name changes…

- I don’t really see enough benefit but also I don’t really care
- I do think the code should either:
  - Introduce a new name for symmetry, and intend to keep the old names forever. Then it’s not really deprecated.
  - Decide to deprecate the old name; in that case this should use the standard `Deprecated:` syntax so that tool make us migrate immediately.

Reply:
… all these name changes apart from the real semantic change WRT `ConfigHome`.

Reply:
I rather not do the unrelated name changes, it just adds churn for no reason. If I look at a code and see the two different functions used I will always have to follow them to figure out if they are actual equal.